container_test.go

```diff
@@ -629,6 +632,71 @@ func TestContainerKill(t *testing.T) {
 	}
 }
 
+func TestKillContainerDeletedByRunc(t *testing.T) {
```

Comment:
There is not common error code for caller from perspective of runtime-spec, which it is burden of containerd to maintain each error message filter. Right now crun is still in integration testing with containerd, I think we can check  `os.Getenv("RUNC_FLAVOR") == "crun"` and skip this case, before we have better solution to handle error.

WDYT?

cc @cpuguy83 @AkihiroSuda

Reply:
Getenv check SGTM

Reply:
OK, fix it soon.